agent/aws.go

```diff
@@ -39,7 +39,7 @@ func awsRegion() (string, error) {
 }
 
 func awsSession() (*session.Session, error) {
-	region, err := awsRegion()
+	region, err := AWSRegion()
```

Comment:
This really belongs somewhere else, but that's not for this PR 🤷‍♂️

Reply:
yeah, agreed - i took a look at it and thought "a problem for another day"